test/mitmproxy/addons/test_view.py

```diff
@@ -147,6 +147,9 @@ def test_create():
         assert v[0].request.url == "http://foo.com/"
         v.create("get", "http://foo.com")
         assert len(v) == 2
+        v.create("get", "http://foo.com\\")
+        v.create("get", "http://")
+        assert len(v) == 2
```

Comment:
Please also assert that a log message was written, e.g., `assert tctx.master.has_log("<check for correct message>", level="error")`